CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_hubspot_source v0.14.1
```

Comment:
I am almost considering that this release should possibly be a breaking `v0.15.0` release.

The addition of a new model could be breaking for users that do not have this in their source. I imagine we should either:
- Keep this as non breaking, but add a new variable for the `merged_deal` and have it be `false` by default so it doesn't break for users.
- Make this a breaking change and also add the new variable, but have the variable be `true` by default.
- Do not create a variable and just make this a breaking change. However, we still run the risk of users upgrading and running into an issue if they do not have the source table.

Reply:
This is a good point that not all users will have `merged_deal`. When thinking through the options, I'm keeping quickstart in mind and therefore I know it's best to be conservative with the addition of new variables. So I think the option that will prevent the most support tickets down the line would be having a variable for the new model which the default is false.

Reply:
updated to default false, thanks for catching